types/shares_test.go

```diff
@@ -1,574 +0,0 @@
-package types
```

Comment:
deleted this because it was moved to celestia-app. However, we likely want to extract a pure code delete PR for the code that was moved from celestia-core to celestia-app

Reply:
Sync'ed with @evan-forbes offline. Created https://github.com/celestiaorg/celestia-core/issues/842